sway-lsp/src/utils/keyword_docs.rs

```diff
@@ -0,0 +1,828 @@
+#![allow(unused)]
+
+use proc_macro2::{TokenStream, TokenTree};
+use quote::ToTokens;
+use std::{collections::hash_map::HashMap, fmt::Write};
+use syn::{parse_quote, ItemMod};
+
+// Ported from https://github.com/rust-lang/rust/blob/master/library/std/src/keyword_docs.rs
+
+/// Documentation for sway keywords.
+/// Primarily used for showing documentation on LSP hover requests.
+#[derive(Debug)]
+pub struct KeywordDocs {
+    /// Key = keyword
+    /// Value = documentation
+    inner: HashMap<String, String>,
+}
```

Comment:
Could we use the same pattern as token_map here?

```suggestion
pub struct KeywordDocs(HashMap<String, String>);
```

Reply:
sure, i'm with that.